src/menu/menu.scss

```diff
@@ -20,11 +20,13 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
   margin: 0;
   padding: 0;
 
-  svg.iui-header-menu-icon {
+  .iui-header-menu-icon {
     margin: 0 $iui-xs;
+    display: inline-flex;
   }
 
   div.iui-header-menu-icon {
+    margin: 0;
```

Comment:
How this one broke?

Reply:
svg selector was removed above, so it was getting `margin: 0 $iui-xs`